sequencer/src/lib.rs

```diff
@@ -184,22 +184,21 @@ impl<N: network::Type> NodeImplementation<SeqTypes> for Node<N> {
     type Storage = Storage;
     type QuorumNetwork = N::QuorumChannel;
     type CommitteeNetwork = N::DAChannel;
-
-    fn new_channel_maps(
-        start_view: ViewNumber,
-    ) -> (ChannelMaps<SeqTypes>, Option<ChannelMaps<SeqTypes>>) {
-        (ChannelMaps::new(start_view), None)
-    }
 }
 
+#[derive(Clone, Debug)]
+pub struct NodeState {}
+impl InstanceState for NodeState {}
+
 impl NodeType for SeqTypes {
     type Time = ViewNumber;
     type BlockHeader = Header;
     type BlockPayload = Payload;
     type SignatureKey = PubKey;
     type Transaction = Transaction;
     type ElectionConfigType = ElectionConfig;
-    type StateType = State;
+    type InstanceState = Instance;
```

Comment:
Should this be `NodeState`? Or what is `Instance`?

Reply:
Yes it should be. Fixed in c958648